test/integration/api/search/search_spec.js

```diff
@@ -83,7 +85,72 @@ describe("search_api", function () {
         with_field: ['context', 'tags']
       });
     });
+
+    describe('to_url', () => {
+      const cloudName = 'test-cloud';
+
+      const commonUrlOptions = {
+        secure: true,
+        cloud_name: cloudName,
+        api_secret: 'test-secret',
+        api_key: 'test-key'
+      };
+
+      beforeEach(() => {
+        cloudinary.v2.config(commonUrlOptions);
+      });
+
+      const search = cloudinary.v2.search.expression('resource_type:image').sort_by('public_id', 'asc').max_results(10);
+
+      const defaultSignature = 'b2eb2ae76343207c92c267130111b241c87c0246';
```

Comment:
Please make sure to use the same values as in the reference implementation, so we get exactly the same results. This will assure that we do not have bugs.